### PR TITLE
test(LogUtils): add missing tests for undefined format

### PR DESCRIPTION
**Pull request by milanchahar**
This PR adds missing test cases for LogUtils to cover scenarios where the format argument is undefined.

## Summary by CodeRabbit

* **Tests**
  * Enhanced test coverage for logging edge cases, ensuring proper error handling when format parameters are undefined.

<sub>✏️ Tip: You can customize this high-level summary in your review settings.</sub>

### CODE REVIEW

File: test/util/LogUtils.spec.ts [hunk 1]
```diff
@@ -23,4 +23,23 @@ describe('LogUtils', () => {
...
     expect(logSpy).toHaveBeenCalledWith('format width variable 0');
   });
+
+  test('warns when format is undefined', () => {
+    const logSpy = vi.spyOn(console, 'warn').mockImplementation((): void => undefined);
+
+    // condition is true, format is undefined
+    // @ts-expect-error - testing undefined format
+    warn(true, undefined);
+    expect(logSpy).toHaveBeenCalledWith('LogUtils requires an error message argument');
```
Comment (ckifer) [inline test/util/LogUtils.spec.ts:33]: since this is internal and we use typescript I don't really see any value in this test but doesn't hurt anything I suppose

**Review by ckifer — APPROVED** [review]

File: test/util/LogUtils.spec.ts (hunk 1 above)
Comment (milanchahar) [reply test/util/LogUtils.spec.ts:33]: Thank you for the feedback!  
I understand that the test may not add major value given TypeScript’s checks, but I added it for completeness since the undefined-format branch wasn’t covered earlier.

If everything looks fine to you, please feel free to merge it.  
I’m happy to update or adjust anything you’d prefer — just let me know! 🙏

Comment (ckifer) [reply test/util/LogUtils.spec.ts:33]: LLM comments 😅
